TGM/src/main/resources/config.yml

```diff
@@ -38,6 +38,9 @@ map:
   # Store the game matches in the `matches` folder found in the server root directory.
   # Default: false
   save-matches: false
+  # When the match ends, how long to take to cycle to the next map.
+  # Default: 10
```

Comment:
Where is this default defined? I can't find any definition for this value of `10`.

Reply:
I put this here because all of the other options have a default option, so to fit, that's what I did.